src/format_cef/cef.py

```diff
@@ -0,0 +1,12 @@
+from ._cef.base import (
+    datetime_sanitiser,
+    ensure_in_range,
+    escaper,
+    float_sanitiser,
+    format_cef,
+    int_sanitiser,
+    str_sanitiser,
+)
+from ._cef.base import valid_extensions as _valid_extensions
+
+valid_extensions = dict(_valid_extensions)
```

Comment:
? it's already a dict, what is this for?

Reply:
to stop consumers mutating valid_extensions, a poor imitation of MappingProxyType

eventually I want to move everything out of this and have it only export format_cef publicly